app/main.py

```diff
@@ -16,4 +16,21 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+# Определяем словарь для хранения всех переменных, сгруппированных по типу
+sorted_variables = {
+    'mutable': [
+        my_favourite_films,
+        marks,
+        collection_of_coins,
+    ],
+    'immutable': [
+        lucky_number,
+        pi,
+        one_is_a_prime_number,
+        name,
+        profile_info,
```

Comment:
The variable `profile_info` is a tuple, which is an immutable data type in Python. However, if the intention is to classify it based on the mutability of its contents, note that tuples can contain mutable objects. In this context, it is correctly placed under 'immutable' since the tuple itself is immutable.